### PR TITLE
Update packages/mineral-ui-tokens/tokens/generic/textColor

### PR DESCRIPTION
**Pull request by tx-steven**
Update grayscale colors for accessibility

### CODE REVIEW

**Review by tx-trent — APPROVED** [review]

File: packages/mineral-ui-tokens/tokens/generic/textColor.json [hunk 1]
```diff
@@ -33,10 +33,10 @@
     { "name": "color_warning_active",  "value": "{!bronze_70}" },
     { "name": "color_warning_focus",   "value": "{!bronze_60}" },
     { "name": "color_warning_hover",   "value": "{!bronze_50}" },
-    { "name": "color_grayscale",         "value": "{!gray_60}" },
-    { "name": "color_grayscalePrimary",  "value": "{!gray_90}" },
-    { "name": "color_grayscale_active",  "value": "{!gray_70}" },
-    { "name": "color_grayscale_focus",   "value": "{!gray_60}" },
-    { "name": "color_grayscale_hover",   "value": "{!gray_50}" }
+    { "name": "color_grayscale",         "value": "{!gray_80}" },
+    { "name": "color_grayscalePrimary",  "value": "{!gray_100}" },
```
Comment (tx-nestor) [inline packages/mineral-ui-tokens/tokens/generic/textColor.json:37]: should this be grayscale_primary? looks like that's how it was but just checking

**Review by tx-nestor — APPROVED** [review]
Approved. I'm assuming these changes are for passing Accessibility failures in WAVE/CodeSniffer?

**Comment by tx-nestor** [comment]
@tx-michael @tx-paola  You may want to verify this isn't going to break TX Print

**Review by tx-michael — APPROVED** [review]

**Comment by tx-michael** [comment]
let's rock n roll! 🎸

**Pull request merged**